content/millennium/dstu2/medications/medication-statement.md

```diff
@@ -9,7 +9,7 @@ title: MedicationStatement | DSTU 2 API
 
 ## Overview
 
-The Medication Statement resource provides a snapshot in time of known medications taken by the patient now or in the past reported by either the patient, significant other or a provider. Future orders are not returned. Documented historical/past/home medications are commonly captured when taking the patient's medical history. Prescriptions without documented compliance are Intended, since we may not know if the patient is actively taking the medication or has filled the prescription. Medications are assumed to be Taken unless documented otherwise. 
+The Medication Statement resource provides a snapshot in time of known medications taken by the patient now or in the past reported by either the patient, significant other or a provider. Future orders are not returned. Documented historical/past/home medications are commonly captured when taking the patient's medical history. Prescriptions without documented compliance are Intended, since we may not know if the patient is actively taking the medication or has filled the prescription. Medications are assumed to be Taken unless documented otherwise.
```

Comment:
What changed on this line?

Reply:
Trailing whitespace was removed from the line.  Not sure why it didn't show up in the diff.  Maybe because the line is so long?